### DISCUSSION TITLE
Error: Nobootable device

### DISCUSSION THREAD

**Discussion by Hablok** [ipxe/ipxe#238] [Q&A] [answered]
I installed the Windows 2003 server on the qemu emulator. After the first shutdown using VNC Viewer I can't get there. At first It started writing that this port is not listenening. Then I entered command qemu-system-i386. After that I was able to connect, but instead of Windows there was written NO BOOTABLE DEVICE. How to fix it? How can I start the system again?

**Comment by NiKiZe** [comment, marked as answer]
My answer in #237 still applies, I can not see how this is related to iPXE? Do you use iPXE during installation, or are you missunderstanding the messages that the VM shows you. (Qemu fallbacks to using iPXE for networkboot when no bootable disk is found)

**Reply by Hablok** [reply]
there was also a link to ipxe.org on the NO BOOTABLE DEVICE screen

**Reply by NiKiZe** [reply]
Yes most likely, as explained, when there is no bootable disk found, qemu by default fallbacks to iPXE, and when all boot options fail you get No bootable device from the BIOS.

My guess is that you don't want to use iPXE at all (maybe don't know what it is or what it does), you just want to boot your disk with qemu - but that is something that you need to ask about in a qemu or win2k3 forum, as it is not related to iPXE.

**Reply by Hablok** [reply]
The disk is virtual and it has not disappeared.  Thanks for the help, I will contact Qemu

**Reply by Hablok** [reply]
Do you know how to find the forum qemu?
